src/views/scroll-editor/scroll-top-toolbar.vue

```diff
@@ -457,10 +459,72 @@ export default class ScrollTopToolbar extends Vue {
 
     }
 
+    private get inTextMode(): boolean {
+        return ('text' === this.scrollEditorState.mode);
+    }
+    private set inTextMode(val: boolean) {
+        const param = 1;
```

Comment:
What is this setter? It does not seem to do anything at all. Why is it here?

Reply:
To avoid runtime warnings :
vue.runtime.esm.js?2b0e:619 [Vue warn]: Computed property "inTextMode" was assigned to but it has no setter.
And if they are regular functions, you get these:
[Vue warn]: Invalid prop: type check failed for prop "pressed". Expected Boolean, got Function for inTextMode() and inMaterialMode()

Reply:
Why is it assigned? Who assigns it? It shouldn't be assigned by anyone.